src/test/groovy/org/jetbrains/intellij/PatchPluginXmlSpec.groovy

```diff
@@ -33,6 +33,23 @@ class PatchPluginXmlSpec extends IntelliJPluginSpecBase {
 """
     }
 
+    def 'patch change notes'() {
+        given:
+        pluginXml << "<idea-plugin version=\"2\"></idea-plugin>"
+        buildFile << "version='0.42.123'\nintellij { version = '14.1.4' }\n"
+        buildFile << "patchPluginXml { changeNotes = 'change notes' }"
+        when:
+        def project = run(IntelliJPlugin.PATCH_PLUGIN_XML_TASK_NAME)
+        then:
+        outputPluginXml(project).text == """<idea-plugin version="2">
```

Comment:
It's a shame that tests don't work for you :( Anyway, the actual result is

```
<idea-plugin version="2">
   <change-notes>change notes</change-notes> 
   <version>0.42.123</version>
   <description>Plugin pluginDescription</description>
   <idea-version since-build="141.1532" until-build="141.*"/>
 </idea-plugin>
```